### PR TITLE
Make adobekey.py work in Python 3

### PR DESCRIPTION
**Pull request by kubik147**
This updates `adobekey.py` to work in Python 3. This is updated from #1152 to apply cleanly to master. Other than #1152, it also drops Python 2 compatibility (no `six`, no `from __future__` imports etc.).

### CODE REVIEW

File: DeDRM_plugin/adobekey.py [hunk 1]
```diff
@@ -112,15 +111,13 @@ def unicode_argv():
             # Remove Python executable and commands if present
```
Comment (kubik147) [inline DeDRM_plugin/adobekey.py:111]: It’s quite possibe that the entire `unicode_argv` function is not necessary anymore. A copy of this function exists in many modules, so this should be changed for all files in a different PR.

**Comment by apprenticeharper** [comment]
Ugree we need to check whether unicode_argv is still needed for Python 3, but needs to be done in Mac, Win and Linux.

**Comment by cclauss** [comment]
#1255 should fix several Python syntax errors (crash bugs) in both Python 2 and Python 3.

The GitHub Action also shows 52 occurrences of builtin functions that have been removed in Python 3 such as file(), raw_input(), reload(), unicode(), xrange().
